packages/terra-detail-view/src/DetailView.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import DetailViewSubtitle from './DetailViewSubtitle';
+import './DetailView.scss';
+
+const propTypes = {
+  /**
+   * The title display for the view
+   **/
+  title: PropTypes.element,
```

Comment:
The idea was that this should accept the title component, but since the react component hasn't been completed for the title yet I just left it open.